src/app/content/courses/create-your-sdk-with-codama/codama-with-anchor/en.mdx

```diff
@@ -0,0 +1,180 @@
+import { ArticleSection } from "../../../../components/ArticleSection/ArticleSection";
+
+# Build your Codama IDL from Anchor IDL
+
+Creating a Codama IDL from Anchor is extremely straightforward. Once you generate your Anchor IDL with anchor build, you can convert it directly using the `createFromRoot` function:
+
+```ts
+import { createFromRoot } from 'codama';
+import { rootNodeFromAnchor } from '@codama/nodes-from-anchor';
```

Comment:
Since `@codama/nodes-from-anchor` is not included in `codama` by default, maybe we should have an installation code snippet for it?